utilities.py

```diff
@@ -236,6 +241,11 @@ def save_roi_signals(
             defs.DoricFile.Attribute.ROI.COORDS:       coords
         }
 
+        if unitIds is not None:
```

Comment:
I'm not sure if this fits well with the following changes in  `names` (line 255), `usernames` (line 258), `dataset_name` (line 261)

Reply:
dataset name has to definitely correspond the id
usernames and names might be not used anywhere and can be deprecated